pkgs/top-level/all-packages.nix

```diff
@@ -14591,6 +14591,18 @@ in
 
   boehmgc = callPackage ../development/libraries/boehm-gc { };
   boehmgc_766 = callPackage ../development/libraries/boehm-gc/7.6.6.nix { };
+  boehmgc_nix = pkgs.boehmgc.override {
+    enableLargeConfig = true;
+  };
+  boehmgc_nixUnstable = pkgs.boehmgc_nix.overrideAttrs (drv: {
+    patches = (drv.patches or []) ++ [
+      # Part of the GC solution in https://github.com/NixOS/nix/pull/4944
+      (pkgs.fetchpatch {
+        url = https://github.com/hercules-ci/nix/raw/5c58d84a76d96f269e3ff1e72c9c9ba5f68576af/boehmgc-coroutine-sp-fallback.diff;
+        sha256 = "sha256-JvnWVTlkltmQUs/0qApv/LPZ690UX1/2hEP+LYRwKbI=";
+      })
+    ];
+  });
```

Comment:
I am not sure if we want this in the top-level. Why not move it into the nix file?

Reply:
I maintain Haskell bindings for Nix and exposing it is useful to me. I guess I could expose it via passthru instead.